src/lib/core/components/computations/plugins/alphaDiv.tsx

```diff
@@ -135,7 +135,7 @@ export function AlphaDivConfiguration(props: ComputationConfigProps) {
       const newComputation = createComputation(
         computation.descriptor.type,
         // @ts-ignore
-        `Data: ${variableObject?.entityDisplayName}: ${variableObject?.displayName}; Method: ${newConfigObject.alphaDivMethod}`,
+        `${variableObject?.entityDisplayName} > ${variableObject?.displayName}&;&${newConfigObject.alphaDivMethod}`,
```

Comment:
Lovely. It's occurred to me that we should also make the same change in the configuration dropdown. So for example changing line 225 in alphaDiv.tsx to something like
```
{collectionVar.entityDisplayName} {'>'} {collectionVar.displayName}
```

Reply:
Good catch! I'll make that change.